config/routes.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.routes.draw do
+  root 'users#new'
+  resources :users, only: %i[new create edit update show]
```

Comment:
Kindly remove the "show" route. It's not part of the requirements for this project.